### DISCUSSION TITLE
JOBRUNR with microservices

### DISCUSSION THREAD

**Discussion by Ghez239** [jobrunr/jobrunr#916] [Q&A]
i have two different recurring jobs in different pod in a kubernetes cluster. Is it possible to use it?
Actually my jobs not starting because the pod n1 try to run the job that is in the pod n2. 
Sorry but i didnt find any information in the documentations.

**Comment by auloin** [comment]
From what I understand, you're trying to launch 2 pods that don't share the same code but use the same database.
- JobRunr Pro's [server tags](https://www.jobrunr.io/en/documentation/pro/server-tags/) is your best option if you can afford the subscription
- Otherwise you need to split your DB or implement _all_ your jobs in the different pods
